src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.macOS.cs

```diff
@@ -61,6 +61,7 @@ protected TextBox(bool isPassword)
 
 		partial void OnTextAlignmentChangedPartial(DependencyPropertyChangedEventArgs e)
 		{
+			UpdateTextBoxView();
```

Comment:
I'm following the other OnXXXChangedPartial here. But I don't understand what's the need to create a new TextBoxView instead of simply updated the changed property?

Reply:
In other platforms, some properties are exclusive, it was easier to update all at once. It may not be needed here, but that's not really a concern either.